javascript/javascript2/week2/doublingNumbers.js

```diff
@@ -0,0 +1,13 @@
+//1.1. Doubling of number
+const numbers = [1, 2, 3, 4];
+checkNumber = i => {
+  return i % 2 != 0
+}
+
+const filteredNumbers = numbers.filter(checkNumber);
+doubleNumber = i => {
+  return i * 2;
+}
+let newNumbers = filteredNumbers.map(doubleNumber);
+
```

Comment:
The solution is right, and you can use chaining method here, so code is shorter 😎

`const doubledOddNumbers = numbers
    .filter((item) => item % 2 !== 0)
    .map((item) => item * 2);
console.log("The doubled numbers are " + doubledOddNumbers);`